spec/models/user_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do  # rubocop:disable all
```

Comment:
これも、もしかしたら以下のようにしたら警告消えるかなと思いました！
```suggestion
RSpec.describe User
```

Reply:
ありがとうございます！
修正案の通り修正したら、警告消えましたのでこちらに書き換えています！